website/docs/extensions/cairo-run.md

```diff
@@ -9,6 +13,32 @@ If not provided, a gas usage is not limited.
 Gas usage can be disallowed by setting the value to `0`.
 The second one prints the full memory after the execution.
 
+## Choosing a function to run
+
+In general, a function to run can be specified in two ways:
+
+- by the name of the function
```

Comment:
nit: I wonder if we should include how to specify this name in this place, to have a nice top-level tldr